bin/run-tests.mjs

```diff
@@ -70,13 +71,14 @@ try {
         } else if (text === `[HARNESS] done`) {
           fulfill();
         } else if (text === `[HARNESS] fail`) {
+          // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
           reject();
         }
       });
-
-      await page.goto('http://localhost:60173?hidepassed&ci');
     })
   );
+
+  await page.goto('http://localhost:60173?hidepassed&ci');
```

Comment:
This is a change in behavior (the old code was fire-and-forget because of the confusing way it used
the `Promise` constructor). This is my best guess, and we need to verify that it still works
correctly.

In general, there's a bunch of "fire and forget" code in the `bin` directory, and we should document
when we do that, and why.